esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3180,7 +3210,7 @@ bool CFileSprayEx::onDropZoneFileSearch(IEspContext &context, IEspDropZoneFileSe
                 IConstTpMachine& tpMachine = tpMachines.item(ii);
                 if (isEmptyString(dropZoneServerReq) || matchNetAddressRequest(dropZoneServerReq, isIPAddressReq, tpMachine))
                 {
-                    searchDropZoneFiles(context, tpMachine.getNetaddress(), dropZone.getPath(), nameFilter, files, filesFound);
+                    searchDropZoneFiles(context, dropZoneName, tpMachine.getNetaddress(), dropZone.getPath(), nameFilter, files, filesFound);
```

Comment:
similar to comments elsewhere, do both dropZoneName and netAddress being set make sense?
If not - it would make things clearer - if only 1 was passed based on mode.

Reply:
yes, but I would prefer code to be kept as similar as possible everywhere, and it is generally better unless time critical to use isContainerized() , so both code paths get compiled and tested.
Code clarity is the most import aspect here/reducing duplicated code etc.

I'd code something like:
```
const char *dropZoneId = isContainerized() ? dropZoneName : tpMachine.getNetaddress();
searchDropZoneFiles(context, dropZoneId, dropZone.getPath(), nameFilter, files, filesFound);
```

Reply:
Thanks for the clarification. I am changing the code related to the getDropZoneScopePermissions(). For searchDropZoneFiles(), I may still need to pass both dropZoneName and netAddress. The dropZoneName is for scope check. The netAddress is for retrieving file information using the SocketEndpoint.